sdks/java/container/build.gradle

```diff
@@ -16,7 +16,11 @@
  * limitations under the License.
  */
 
-plugins { id 'org.apache.beam.module' }
+plugins {
+  id 'org.apache.beam.module'
+  id 'com.github.jk1.dependency-license-report' version '1.13'
```

Comment:
Kenn mentioned hierynomus/license-gradle-plugin found  more accurate license dependencies; does that need to be added?

Reply:
If I understand it correctly, dependencies without licenses/notice included in jar are *considered* as the missing dependencies at first. However, the reality is dependency scan is correct, it only missing licenses.  `com.github.jk1.dependency-license-report` generates `index.json` file as well, which provides a list of all dependencies. I used this file to go through all dependencies and pull licenses/notices if they are not pulled automatically.